marlowe-dashboard-client/tailwind.config.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = {
+  purge: [],
+  darkMode: false, // or 'media' or 'class'
+  theme: {
```

Comment:
Something I thought we may want to do, is to be very consius of what plugis we use and how we customize them. For that reason, we maaaaaaay want to disable all plugins except the ones we'll use, and customize the ones we use with the values we know we are gonna use. That way we don't end up with padding-96

https://tailwindcss.com/docs/theme#disabling-an-entire-core-plugin

What do you think?